lib/matplotlib/axes/_base.py

```diff
@@ -3132,6 +3132,35 @@ def set_navigate_mode(self, b):
         """
         self._navigate_mode = b
 
+    def save_view(self):
+        """
+        Save information required to reproduce the current view.
+
+        Called before a view is changed, such as during a pan or zoom
+        initiated by the user.
+
+        .. note::
+
+            Intended to be overridden by new projection types, but if not, the
+            default implementation saves the view limits. You *must* implement
+            :meth:`restore_view` if you implement this method.
+        """
+        raise NotImplementedError
```

Comment:
It would be simpler to put the existing code for ordinary axes here instead of leaving these as NotImplementedError, wouldn't it?  I don't see any disadvantage, but I'm probably missing something.